updated README for Nessus (#271)
RedHatProductSecurity/rapidast: README.md

RapiDAST provides:

- Automated HTTP/API security scanning using ZAP

> - Kubernetes opearator scanning using OOBTKUBE

Reviewer: ```suggestion
- Kubernetes operator scanning using OOBTKUBE
```

> - Automated vulnerability scanning using Nessus (requires a Nessus license and instance)

Reviewer: ```suggestion
- Automated vulnerability scanning using Nessus (requires a Nessus instance)
```

- Command-line execution with yaml configuration, suitable for integration in CI/CD pipelines
- Ability to run automated DAST scanning with pre-built or custom container images
- HTML, JSON and XML report generation
- Integration with Google Cloud Storage and [OWASP DefectDojo](https://owasp.org/www-project-defectdojo/)

> RapiDAST is to be used for testing purpose, and should not be used on production systems.

Reviewer: ```suggestion
RapiDAST is for testing purposes, and should not be used on production systems.
```
